Calculator/Calculator/Controller/ViewController.swift

```diff
@@ -5,12 +5,228 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    
+    private let numberFormatter = NumberFormatter()
+    private var inputs: String = ""
+    private var currentOperand: String = ""
+    private var currentOperator: String = ""
+    private var isCalculated: Bool = false
+    private var isFirstInput: Bool {
+        return self.stackViewInScrollView.subviews.isEmpty
+    }
+    
+    @IBOutlet private weak var scrollView: UIScrollView!
+    @IBOutlet private weak var stackViewInScrollView: UIStackView!
+    @IBOutlet private weak var currentOperandLabel: UILabel!
+    @IBOutlet private weak var currentOperatorLabel: UILabel!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        setNumberFormatter()
+        self.currentOperatorLabel.text = ""
+        self.currentOperandLabel.text = "0"
+    }
+    
+    @IBAction private func didTapButton(sender: UIButton) {
+        guard let buttonTitle = sender.currentTitle else { return }
+        
+        switch buttonTitle {
+        case "=":
+            calculate()
+        case "AC":
+            clearAll()
+        case "CE":
+            clearCurrentOperand()
+        case "⁺⁄₋":
+            reverseOperand()
+        case "÷", "×", "−", "+":
+            clickedOperator(buttonTitle)
+        case "0", "00":
+            addZeroToOperandLabel(buttonTitle)
+        case ".":
+            addDotToOperandLabel()
+        default:
+            addNumberToOperandLabel(buttonTitle)
+        }
+    }
+    
+    func setNumberFormatter() {
+        self.numberFormatter.numberStyle = .decimal
+        self.numberFormatter.roundingMode = .halfUp
+        self.numberFormatter.usesSignificantDigits = true
+        self.numberFormatter.maximumSignificantDigits = 20
+    }
+    
+    func calculate() {
+        if self.isCalculated == false {
+            if self.currentOperand != "" {
+                addStackView(number: self.currentOperandLabel.text, operatorType: self.currentOperator)
+            }
```

Comment:
`""`보다는 `isEmpty`가 의미를 나타내기 좋겠네요 🙂
```suggestion
            if self.currentOperand.isEmpty == false {
                addStackView(number: self.currentOperandLabel.text, operatorType: self.currentOperator)
            }
```